File: MCP服务器/finance/QuantConnect_mcp-server_3a656b0_python/tests/algorithms/live_charts.py
# region imports
from AlgorithmImports import *
# endregion


class LiveChartTestAlgorithm(QCAlgorithm):

    def initialize(self):
        symbol = self.add_crypto("BTCUSD", Resolution.SECOND).symbol
        self.plot_indicator("SMA", self.sma(symbol, 10))

File: reference-mcp-servers/finance/QuantConnect_mcp-server_3a656b0_python/tests/algorithms/LiveCharts.cs
#region imports
    using QuantConnect;
    using QuantConnect.Algorithm;
#endregion

public class LiveChartTestAlgorithm : QCAlgorithm
{
    public override void Initialize()
    {
        var symbol = AddCrypto("BTCUSD", Resolution.Second).Symbol;
        PlotIndicator("SMA", SMA(symbol, 10));
    }
}
